extensions/scarb-doc/src/docs_generation/markdown.rs

```diff
@@ -26,6 +26,12 @@ const BASE_HEADER_LEVEL: usize = 1;
 const SOURCE_DIRECTORY: &str = "src";
 const BOOK_TOML_FILENAME: &str = "book.toml";
 pub const SUMMARY_FILENAME: &str = "SUMMARY.md";
+const SHORT_DOCUMENTATION_LEN: usize = 200;
```

Comment:
changed from 100 to 200 because it seemed too short when `SHORT_DOCUMENTATION_LEN` became a strict top limit in `fn get_short_documentation`